bioindex/lib/index.py

```diff
@@ -241,7 +243,7 @@ def delete_stale_keys(self, engine, objects, console=None):
 
                 # delete stale or missing keys
                 for kid in updated_or_deleted_files:
-                    sql = f'DELETE FROM {self.table.name} WHERE `key` = :key'
+                    sql = f'DELETE FROM `{self.table.name}` WHERE `key` = :key'
```

Comment:
This change is necessary so that the sql can handle tables after uuids like they are in the "minibioindex"